css/properties/using-css-variables.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This kind of page doesn't get it's own table. I think another one should be reused, but I'm not sure which one.

Reply:
Not sure how to proceed here. Looping in @chrisdavidmills. Chris, is this the correct thing to do here?

Reply:
Hrm, this is an interesting one. CSS variables/custom properties aren't a property, and they also aren't really a type, at-rule or selector. I'd be inclined to call the page "css_custom_properties.json" (use the official name), and put it in the root of the css directory, in the same sort of way we do for html/global_attributes.json.

For the MDN URL, I think the "Using CSS variables" page is still OK — this is the best thing we've got for now.

I also don't really understand why this is lumped in with the block-size/inline-size data changes. They should probably be separated out into two PRs?

@Elchi3 , does this seem like a reasonble plan to you, for including CSS variables in BCD?

Reply:
I think we've already migrated https://developer.mozilla.org/en-US/docs/Web/CSS/--*#Browser_compatibility to https://github.com/mdn/browser-compat-data/blob/master/css/properties/custom-property.json. 

I don't think we need to add it twice. https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables should probably also use "css.properties.custom-property".

Reply:
Ah, so we have already got it. Thanks Florian!

So @cartinez , remove the using-css-variables.json from the PR, and just update the guide page with the same macro call used by https://developer.mozilla.org/en-US/docs/Web/CSS/--*#Browser_compatibility.

Reply:
Will do! Also sorry for mixing up the other JSONs in this PR, this is my first contribution ever so I'm still learning.

Reply:
@cartinez no worries at all. We are really grateful for your contribution.

Reply:
Do I update the PR title too or is it ok like this?

Reply:
It probably doesn't matter all that much at this stage, but strictly speaking something like "update compat data for inline-size/block-size" would be good.

Reply:
@jpmedley it's ok, I just felt a little overwhelmed as it was my first contribution but I realize it's an edge case. I was looking at the CSS variables page and noticed the announcement on that table so I decided to jump in, I learned something new anyway.

Reply:
We appreciate the help.